### DISCUSSION TITLE
Python Launcher version action

### DISCUSSION THREAD

**Discussion by oleksis** [brettcannon/python-launcher#259] [Ideas]
Ability to return the version of Python launcher running

- Current behavior: Returning only the `python` version
```bash
➜ py -3.10 --version
Python 3.10.9
```

- Expected behavior: Return  the `py` version
```bash
➜ py --version
Python Launcher for Unix 1.0.0
Python 3.10.9
```

**Comment by oleksis** [comment]
Follow this https://github.com/oleksis/cpython/commit/79a281ffdd38c20182a71b2d706bad8e6b351a11 commit to the Python Launcher for Windows (API)

Behavior like:
```pwsh
➜ $env:PY_PYTHON='3.10'
 
➜ .\PCbuild\amd64\py_d.exe -V
Python launcher for Windows 3.11.1+
Python 3.10.9

➜ .\PCbuild\amd64\py_d.exe --version
Python launcher for Windows 3.11.1+
Python 3.10.9

➜ .\PCbuild\amd64\py_d.exe -V:3.10 -V
Python 3.10.9

➜ .\PCbuild\amd64\py_d.exe -V:3.10 --version
Python 3.10.9
```

Any adivice for contribution this to CPython upstream ?

**Reply by oleksis** [reply]
For keep backward-compatibility and use cases, we can use the `--version-launcher` option https://github.com/python/cpython/issues/100829#issuecomment-1374650747

**Reply by brettcannon** [reply]
Going to keep the conversation focused in https://github.com/python/cpython/issues/100829 for wider input.
